README.md

```diff
@@ -10,17 +10,15 @@
 Repository to describe, develop, document, and test the Tenure API within the KnowYourCustomer (KYC) API family.
 
 ## Scope
-* Service API(s) for “Tenure” (see APIBacklog.md)
+* Service API(s) for “Tenure” (see [APIBacklog.md](https://github.com/camaraproject/APIBacklog/blob/main/documentation/APIbacklog.md))
 * It provides the customer with the ability to:  
-  * (Please update with the scope of the API)
-  <!-- delete the following line if the scope of the API is defined differently in APIBacklog -->
-  * NOTE: The scope of this API family should be limited (at least at the first stage) to 4G and 5G.  
+  * Establish a level of trust for a given network subcription identifier (e.g. MSISDN) by verifying the period of time (i.e. "tenure") for which the end user currently allocated that identifier has been associated with it by their current network operator.
```

Comment:
Can we use phone number here instead of MSISDN? Or is there a specific reason that it has to be an MSISDN?

```suggestion
  * Establish a level of trust for a given network subcription identifier (e.g. phone number) by verifying the period of time (i.e. "tenure") for which the end user currently allocated that identifier has been associated with it by their current network operator.
```